lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -43,19 +44,30 @@ function resolveLocalOrCwd(payloadPath) {
  * Launch Chrome and do a full Lighthouse run.
  * @param {string} url
  * @param {string} configPath
+ * @param {boolean=} isDebug
  * @return {!LighthouseResults}
  */
-function runLighthouse(url, configPath) {
+function runLighthouse(url, configPath, isDebug) {
   const command = 'node';
   const args = [
     'lighthouse-cli/index.js',
     url,
     `--config-path=${configPath}`,
+    `--output-path=smokehouse.report.json`,
```

Comment:
I often run `yarn smokehouse`  and `yarn smoke` simultaneously which could conflict in this scenario. 

also my smokehouse revamp PR means we'll have a lot of these running at the same time.

how about something like:  
```js
const filename = `smokehouse${Math.round(Math.random()*1000))}.report.json`;
```